sharding/sharding.go

```diff
@@ -119,8 +119,11 @@ func (r *ShardingFerry) Run() {
 	cutoverStart := time.Now()
 	// The callback must ensure that all in-flight transactions are complete and
 	// there will be no more writes to the database after it returns.
-	metrics.Measure("CutoverLock", nil, 1.0, func() {
-		err = r.config.CutoverLock.Post(client)
+	err = ghostferry.WithRetries(r.config.MaxCutoverRetries, time.Duration(r.config.CutoverRetryWaitSeconds)*time.Second, r.logger, "get cutover lock", func() (err error) {
```

Comment:
`time.Duration(r.config.CutoverRetryWaitSeconds)*time.Second` is confusing. 

`time.Duration` represents nano seconds but it _happens_ to also be an `int64` which works out at the end here. 

I believe what you want is `r.config.CutoverRetryWaitSeconds*time.Second`

Reply:
We need to cast it to `time.Duration` before attempting the operation as this change gives:
```sh
# github.com/Shopify/ghostferry/sharding
./sharding.go:122:91: invalid operation: r.config.Config.CutoverRetryWaitSeconds * time.Second (mismatched types int and time.Duration)
```

Reply:
ah i see, 👍